benchmark.py

```
@@ -0,0 +1,833 @@
+import aiohttp
+import asyncio
+import time
+import random
+import argparse
+import msgpack
+import subprocess
+import shutil
+import os
+from pathlib import Path
+import math
+from typing import Dict, List, AsyncGenerator, Tuple, Any, NamedTuple
```

Ruff (0.12.2)

Import from `collections.abc` instead: `AsyncGenerator`

Import from `collections.abc`

(UP035)